packages/schema/lib/functional-constraints/AuthFieldisSafe.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const jsonschema = require('jsonschema');
+
+const AUTH_FIELD_ID = '/AuthFieldSchema';
+const AUTH_FIELDS_ID = '/AuthFieldsSchema';
+
+const FORBIDDEN_KEYS = [
+  'access_token',
+  'api_key',
+  'apikey',
+  'api-key',
+  'auth',
+  'jwt',
+  'passwd',
+  'password',
+  'pswd',
+  'refresh_token',
+  'secret',
+  'set-cookie',
+  'signature',
+  'token',
+];
+
+const isSensitiveKey = (key = '') =>
+  FORBIDDEN_KEYS.some((forbidden) =>
+    key.toLowerCase().includes(forbidden.toLowerCase()),
```

Comment:
```suggestion
    key.toLowerCase().includes(forbidden),
```

Already in lower case